content/en/tracing/setup_overview/setup/go.md

```diff
@@ -144,36 +144,35 @@ Install and configure the Datadog Agent to receive traces from your now instrume
 
 3. After the application is instrumented, the trace client attempts to send traces to the Unix domain socket `/var/run/datadog/apm.socket` by default. If the socket does not exist, traces are sent to `http://localhost:8126`.
 
-   If a different socket, host, or port is required, use the `DD_TRACE_AGENT_URL` environment variable. Some examples:
+   A similar rule applies to all metrics sent by the Go tracer (including Runtime Metrics and internal telemetry): the client will attempt to send Dogstatsd data to the Unix domain socket `/var/run/datadog/dsd.socket` and will default to `http://localhost:8125` if that does not exist.
+
+   If different hosts or ports are required, one or more of the below environment variables can be used. The examples show the defaults but can be set to other values as well.
 
    ```
-   DD_TRACE_AGENT_URL=http://custom-hostname:1234
-   DD_TRACE_AGENT_URL=unix:///var/run/datadog/apm.socket
+   DD_AGENT_HOST=localhost   # the default host to send traces and metrics too
```

Comment:
Since **DD_AGENT_HOST** defaults to localhost when not set, should 

```
DD_AGENT_HOST=localhost   # the default host to send traces and metrics too
```

be something like:

```
DD_AGENT_HOST=localhost   # the host to send traces; defaults to `localhost` when not set
```

?

And should the description note this is for traces only, while metrics follow dogstatsd?

Reply:
It's the agent host. For both.

Reply:
Okay! I'll resolve this comment.